docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -23,61 +27,53 @@ VMs might fail to start because no boot source is available for cloning.
 
 ## Diagnosis
 
-1. Check the cluster for a default storage class:
+1. Check the cluster for a default Kubernetes storage class:
+   ```bash
+   $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubernetes\.io\/is-default-class=="true")].metadata.name}'
+   ```
 
+   Check the cluster for a default virtualization storage class:
    ```bash
-   $ kubectl get sc
+   $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubevirt\.io\/is-default-virt-class=="true")].metadata.name}'
    ```
 
-   The output displays the storage classes with `(default)` beside the name of
-   the default storage class. You must set a default storage class, either on
-   the cluster or in the `DataImportCron` specification, in order for the
-   `DataImportCron` to poll and import golden images. If no storage class is
-   defined, the DataVolume controller fails to create PVCs and the following
-   event is displayed: `DataVolume.storage spec is missing accessMode and no
-   storageClass to choose profile`.
+   The output displays the default (Kubernetes and/or virtualization) storage
+   class. You must either set a default storage class on the cluster, or ask for
+   a specific storage class in the `DataImportCron` specification, in order for
+   the `DataImportCron` to poll and import golden images. If the default
+   storage class does not exist, the created import DataVolume and PVC will be
+   in `Pending` phase.
 
-2. Obtain the `DataImportCron` namespace and name:
+2. Obtain the `DataImportCrons` which are not up-to-date:
 
    ```bash
-   $ kubectl get dataimportcron -A -o json | jq -r '.items[] | select(.status.conditions[] | select(.type == "UpToDate" and .status == "False")) | .metadata.namespace + "/" + .metadata.name'
+   $ kubectl get dataimportcron -A -o jsonpath='{range .items[*]}{.status.conditions[?(@.type=="UpToDate")].status}{"\t"}{.metadata.namespace}{"/"}{.metadata.name}{"\n"}{end}' | grep False
    ```
 
 3. If a default storage class is not defined on the cluster, check the
-`DataImportCron` specification for a default storage class:
+`DataImportCron` specification for optional storage class:
```

Comment:
if defined in the `DataImportCron` spec it's "default" anymore.

Reply:
It means that when one specifies storage class in the `DataImportCron` spec DataVolume template it is used instead of the cluster default storage class.